Scripts/MovingLaser.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MovingLaser : MonoBehaviour {
+
+    private Vector3 startPosition;
+
+    [SerializeField]
+    private float moveLeftDistance;
+
+    [SerializeField]
+    private float speed;
+
+
```

Comment:
remove extra white space (single blank line separating field from function is plenty)